Sources/Library/Models/User.swift

```diff
@@ -52,11 +52,11 @@ public final class User: Model {
     /// Internal: Fluent::Model::init(Node, Context).
     public init(node: Node, in context: Context) throws {
         self.id = try? node.extract("id")
-        self.career_account = try node.extract("career_account")
-        self.first_name = try node.extract("first_name")
-        self.last_name = try node.extract("last_name")
+        self.career_account = try? node.extract("career_account")
```

Comment:
Using optionals is pretty shady business, but I'll take it because it seems to work...